crates/pixi_glob/Cargo.toml

```diff
@@ -15,9 +15,10 @@ itertools = { workspace = true }
 memchr = { workspace = true }
 rattler_digest = { workspace = true }
 thiserror = { workspace = true }
-tokio = { workspace = true, features = ["sync"] }
+tokio = { workspace = true, features = ["sync", "rt"] }
```

Comment:
What do we need `rt` for?

Reply:
I get an error for `spawn_blocking` when running `cargo t` in the `pixi_glob` folder otherwise, unsure why this did not occur before? https://docs.rs/tokio/latest/tokio/task/fn.spawn_blocking.html

Reply:
Ah ok, but then we should only add them for `dev-dependencies`.